src/effects/effectchainslot.cpp

```diff
@@ -44,6 +44,10 @@ EffectChainSlot::EffectChainSlot(EffectRack* pRack, const QString& group,
     m_pControlChainSuperParameter->set(0.0);
     m_pControlChainSuperParameter->setDefaultValue(0.0);
 
+    m_pControlChainSuperForceUpdate = new ControlObject(ConfigKey(m_group, "super_force_next_update"));
```

Comment:
> OK, you set "meta_force_next_update" to 1 and then the next touch of the meta knob will let the effect parameter jump?

Not exactly. The next signal received from the controller is the initial state of the knob, not the first time it is turned by the user.

>  Or do you have an Init script that updates all Mappings, can the controller do this?

It will be up to controller mappings to do this if they send a message to the controller requesting the controller to send the state of all the knobs on startup. The EffectUnit.init() function in the library takes care of this. I will make it take a boolean parameter to disable that behavior if it is not appropriate for the controller.

> Why should this only happen for Controllers and not for the GUI?

Because I will add to #1092 after this is merged to make the state of the meta and superknobs persistent. The force_next_update COs are for the case when a controller is used that can send the state of all its knobs without having to turn them first. It is likely that the knobs will move between the time a DJ unplugs it from their computer and plugs it in at a gig. In this case, as described in #1103, it is important that there are no surprises about how the effects sound.

> What is the difference between softtakover in the mapping and this one here?

I'm not quite sure what you're asking. engine.softTakeover() is used, for example, by the library when switching between a focused effect and controlling metaknobs. This prevents sudden jumps in the CO value. Even though the usual soft takeover between metaknobs and parameters would prevent a jump in the sound in case a metaknob jumps to a different value, it would be confusing because the metaknob on screen would move but there would not be a change in sound. So, the library activates engine.softTakeover() for the metaknob.

> Dos it make sense to disable softtakover if the effect is disabled?

Interesting question. I'll play with that. My guess is that it would make sense.

>  This might be coupled with the "Enabled" control that will be off by default (your other PR)

Yeah, if the soft takeover is disabled when the effect is off the force_next_update COs will be unnecessary.

> How about introducing a global "enable_softtakover" co switch? This could be disabled during the initing the controller?

I'm not sure what value that would add.

> I do not know how fast we will get a conclusion here, so you might want to move the "meta_force_next_update" changes to a separate pull request.

I think all these changes should be merged together.